### PR TITLE
Release/0.3.0

### PR DESCRIPTION
**Pull request by jbeemster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Assuming you already have [SBT 0.13.0] [sbt] installed:
 The Kinesis S3 LZO Sink has the following command-line interface:
 
 ```
-snowplow-lzo-s3-sink: Version 0.2.1. Copyright (c) 2014, Snowplow Analytics
+snowplow-kinesis-s3: Version 0.3.0. Copyright (c) 2015, Snowplow Analytics
```
Comment (alexanderdean) [inline README.md]: Copyright notice should range over years of release, so 2014-2015

Comment (jbeemster) [inline README.md]: No worries, will make do that through all of the files.

File: src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/package.scala [hunk 2]
```diff
@@ -26,14 +26,24 @@ import Scalaz._
...
    */
-  type EmitterInput = (String, Validation[List[String], Array[Byte]])
+  type EmitterInput = Validation[FailedRecord, RawRecord]
+
+  /**
+   * Case class to contain a Failed Record:
+   * - The list of errors
+   * - The original line, Base64 Encoded
+   */
+  case class FailedRecord(errors: List[String], line: String)
```
Comment (alexanderdean) [inline src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/package.scala]: Can we define this earlier in the file, before the type that uses it?

File: src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/LzoSerializer.scala [hunk 3]
```diff
@@ -83,18 +86,21 @@ object LzoSerializer {
...
+      Success(record) <- records 
+    } yield {
+      try {
+        rawBlockWriter.write(record)
+        record.success
       } catch {
         case e: IOException => {
           log.warn(e)
-          (record._1, List("Error writing raw event to output stream: [%s]".format(e.toString)).fail)
+          val base64Record = new String(Base64.encodeBase64(record))
```
Comment (alexanderdean) [inline src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/LzoSerializer.scala]: Missing encoding for new String, always set the encoding.

File: src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/SinkApp.scala [hunk 4]
```diff
@@ -43,7 +50,7 @@ object SinkApp extends App {
   val parser = new ArgotParser(
     programName = "generated",
     compactUsage = true,
-    preUsage = Some("%s: Version %s. Copyright (c) 2013, %s.".format(
+    preUsage = Some("%s: Version %s. Copyright (c) 2015, %s.".format(
```
Comment (alexanderdean) [inline src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/SinkApp.scala]: 2014-2015

File: src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/SinkApp.scala [hunk 5]
```diff
@@ -68,19 +75,34 @@ object SinkApp extends App {
 
   val conf = config.value.getOrElse(throw new RuntimeException("--config argument must be provided"))
 
+  val tracker = if (conf.hasPath("enrich.monitoring.snowplow")) {
+    SnowplowTracking.initializeTracker(conf.getConfig("enrich.monitoring.snowplow")).some
+  } else { 
+    None
+  }
```
Comment (alexanderdean) [inline src/main/scala/com.snowplowanalytics.snowplow.storage.kinesis/s3/SinkApp.scala]: Nice

**Comment by alexanderdean** [comment]
Looks great - very minor comments

File: README.md (hunk 1 above)
Comment (jbeemster) [inline README.md]: So should it be 2014-2015 in the top of each file?

Comment (alexanderdean) [inline README.md]: Yep please

**Pull request merged**